engine/reference/builder.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Dockerfile reference
+description: "Dockerfiles use a simple DSL which allows you to automate the steps you would normally manually take to create an image."
+keywords: build, dockerfile, reference
+toc_max: 3
+redirect_from:
+- /reference/builder/
+---
+
+{% github_content site.buildkit.repo site.buildkit.ref site.buildkit.default_branch "frontend/dockerfile/docs/reference.md" %}
```

Comment:
ok found a nice way to inject remote content using a custom plugin implementing a liquid tag. Let me know what you think about it @thaJeztah

Reply:
Here are the logs:

```
7:46:43 AM: Starting plugin github_content.rb...
7:46:43 AM:   Inject https://raw.githubusercontent.com/crazy-max/buildkit/docs-ref-move-frontmatter/frontend/dockerfile/docs/reference.md to engine/reference/builder.md
7:46:43 AM:     edit_url:  https://github.com/crazy-max/buildkit/edit/master/frontend/dockerfile/docs/reference.md
7:46:43 AM:     issue_url: https://github.com/crazy-max/buildkit/issues/new?body=File: [frontend/dockerfile/docs/reference.md](https://docs.docker.com/engine/reference/builder.md)
7:46:43 AM: done in 0.105188682 seconds
```